IdentityCore/src/cache/accessor/MSIDLegacyTokenCacheAccessor.m

```diff
@@ -365,6 +385,13 @@ - (MSIDLegacySingleResourceToken *)getSingleResourceTokenForAccount:(MSIDAccount
 - (BOOL)validateAndRemoveRefreshToken:(MSIDBaseToken<MSIDRefreshableToken> *)token
                               context:(id<MSIDRequestContext>)context
                                 error:(NSError **)error
+{
+    return [self validateAndRemoveRefreshableToken:token context:context error:error];
```

Comment:
why an abstraction here with simply passing along every input param coming into next and does nothing else?

Reply:
Now I added `validateAndRemovePrimaryRefreshToken` which will use the above abstraction as well :)